internal/convert/cluster_config.go

```diff
@@ -134,17 +199,12 @@ func patchReplicaSet(out *opsmngr.AutomationConfig, newReplicaSet *opsmngr.Repli
 	out.ReplicaSets[pos] = newReplicaSet
 }
 
-// protocolVer determines the appropriate protocol based on FCV
-// return "0" for versions <4.0 or "1" otherwise
-func protocolVer(version string) (string, error) {
-	ver, err := semver.NewVersion(version)
-	if err != nil {
-		return "", err
-	}
-	constrain, _ := semver.NewConstraint(fcvLessThanFour)
-
-	if constrain.Check(ver) {
-		return zero, nil
+func patchSharding(out *opsmngr.AutomationConfig, s *opsmngr.ShardingConfig) {
```

Comment:
So this doesn't modify existing sharding configs -- is that not supported? Might be worth putting a comment for the function

Reply:
I just didn't want to add test for this and manual test takes all my resources, I've added a TODO with a CLOUDP to test it later